src/app/home/directives/home-campaign.component.css

```diff
@@ -0,0 +1,67 @@
+h2 {
```

Comment:
in creating this, did you run into anything we should extract into the styleguide for those cards component tickets? just curious if we could get more specific with those tickets or the responsive grid ticket.